### DISCUSSION TITLE
Getoutline container is not up

### DISCUSSION THREAD

**Discussion by nestorbird** [outline/outline#5246] [General]
Hi, 

I'm trying to install getoutline on my server but i'm not able to as i'm getting errors.

1. My getoutline container is not UP getting this error.
yarn run v1.22.18
$ node ./build/server/index.js
{"level":"warn","message":"Environment configuration is invalid, please check the following:\n\n"}{"level":"warn","message":"- SECRET_KEY's byte length must fall into (32, 64) range"}
error Command failed with exit code 1.

I have already created a key with this command openssl rand -hex 32 and entered in .env.sample file.

2. steveltn/https-portal
    minio/minio.
    
These two containers are in restarting state. |

**Comment by bmm-alc** [comment]
perhaps I'm wrong, but are you sure the filename should not be `.env.` instead of `.env.sample`.
